[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute (#629)
MicrosoftDocs/office-developer-client-docs: README.md

This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

> The live docs for this repo can be accessed at [Office client development](/office/client-developer/office-client-development).

Reviewer: ```suggestion
Access the [Office client development](/office/client-developer/office-client-development) documentation on Microsoft Docs.
```
